cmd/config/cluster-maximums/cluster-maximums.yml

```diff
@@ -0,0 +1,141 @@
+# Meant to be run on a 24 node cluster with max-pods per node configured to 2000
+# Fills with pods the worker nodes. Creates 1 routes and service per 4 pods
+global:
+  gc: {{.GC}}
+  indexerConfig:
+    esServers: ["{{.ES_SERVER}}"]
+    insecureSkipVerify: true
+    defaultIndex: {{.ES_INDEX}}
+    type: {{.INDEXING_TYPE}}
+jobs:
+  - name: pods-per-node
+    namespace: pods-per-node
+    jobIterations: {{.JOB_ITERATIONS}}
+    qps: {{.QPS}}
+    burst: {{.BURST}}
+    namespacedIterations: true
+    iterationsPerNamespace: 50
```

Comment:
There are several dimensions here and I applaud you taking the initiative on codifying a max pods workload.

I am wondering whether this is the right value. It will result in namespaces capped at 50 iterations which breaks down into: 50x svcs, 50x server deployments, 100x server pods , 50x client deployments, 100x client pods.

If I were to propose this be a flag, then I think we would end up with flag explosion, so I may not make that proposal if we want to be opininoated.

I am curious whether we intend for these namespaces to be pseudo-realistic, or to target an actual maximum? (e.g. 6k services, which admittedly isn't an actual pod limitation).